frontend/src/routes/users/me/+page.svelte

```diff
@@ -1,71 +1,169 @@
 <script lang="ts">
   import Icon from '$components/icons/Icon.svelte';
   import { put } from '$utils/fetch';
-  let birthdate: string;
-  let isSuccess = false;
+  import type { PageData } from './$types';
+  export let data: PageData;
 
-  const submitBirthdate = async () => {
-    const response = await put('/v1/users/me', { birthdate });
+  type OperationStatus = 'success' | 'failure' | 'incompleted';
 
-    if (response) {
-      isSuccess = true;
-    }
+  let firstName = data.response.first_name;
+  let lastName = data.response.last_name;
+  let email = data.response.email;
+  let birthdate = data.response.birthdate;
```

Comment:
You could use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here. (I'm 98% confident this will work cos of Svelte updates, but check anyway)

```ts
const { firstName, lastName, email, birthdate } = data.response
```